main/getting-started/intro-zoe.md

```diff
@@ -0,0 +1,279 @@
+const newInviteP = outcome;
+const inviteIssuer = zoe.getInviteIssuer();
+const bobExclusiveInvite = await inviteIssuer.claim(newInviteP);
+const instanceHandle = await getInstanceHandle(bobExclusiveInvite);
+
+const {
+  installationHandle: bobInstallationId,
+  issuerKeywordRecord: bobIssuers,
+} = zoe.getInstance(instanceHandle);
+```
+
+
+Bob decides to make his own offer, which happens to match up with Alice’s offer (assume his payments were constructed similar to Alice’s);
+```const bobProposal = harden({
+  want: { Asset: moola(3) },
+  give: { Price: simoleans(7) },
+  exit: { onDemand: null },
+})
+
+
+// Bob escrows with zoe and makes an offer
+const { outcome: bobOfferResult, payout: bobPayoutP } = await E(zoe).offer(
+  bobExclusiveInvite,
+  bobProposal,
+  bobPayments,
+);
+```
+Bob has also gotten back an *outcome* and a *promise* for a *payout*.
+
+
+## Satisfying and completing offers
+
+
+At this point, both the offers that Alice and Bob made are known to the contract instance. The Atomic Swap contract code determines that they are matching offers.  The contract instance calls `reallocate`, which *reallocates* the amounts which are the accounting records within Zoe. Payouts are not created here. 
+
+
+The contract instance then *completes* the offers. A call to `zcf.complete()` with both offers as arguments, makes the payouts to the offer holders.  This takes the amounts from the account records, and withdraws the amounts specified by the offers from the digital assets escrowed within Zoe. It's only at the `complete` step that *amounts* are turned into real payments. This is when the payout *promise* resolves into payments. 
+
+
+The offers are now completed, and nothing more is or can be done with them. Zoe deletes completed offers from the contract instance.
+
+
+## Auction example
+
+
+Let’s look at a more complicated example: an auction, where many users might make a bid offer for something, but only one offer will win the auction. Assuming everything validated, Zoe would give the holder of the auctioned item the escrowed amount from the winning offer, and give the holder of the winning offer what they wanted, the item that had been up for bid.
+
+
+But what about all those other bidders who escrowed their bid amounts with Zoe and didn’t win? Zoe guarantees they all get a *refund* of their escrowed amount. Zoe’s *offer safety* guarantees offer makers either get what they wanted or get refunded whatever they put in Zoe’s escrow with their offer.
+
+
+It’s even possible to get both what you wanted in an offer, and at least a partial refund of what you offered. Consider an auction where you make a bid offer where you’re willing to pay up to 10 quatloos, but if you can win with a lower bid, that’s your offer. You win the auction, but your winning bid amount is only 8 quatloos. The payout would resolve to both the item up for auction you get as the winning bidder and the refund of the 2 extra quatloos you escrowed in the offer. 
```

Comment:
Perhaps not all readers are used to this idea, so we should make it explicit:

"Consider an auction where your bid specifies your maximum payment, but the auctioneer charges just enough to beat the next highest bidder."  Or you could refer to a "second price auction", in which the winner pays the amount bid by the second highest bid. This is also known as a "Vickrey Auction", and you could link to wikipedia for an explanation.